src/rtString.cpp

```diff
@@ -91,10 +91,14 @@ void rtString::term()
 
 void rtString::append(const char* s) 
 {
-  size_t sl = strlen(s);
-  size_t dl = strlen(mData);
```

Comment:
Wasn't it enough to modify that to:  `size_t dl = mData ? strlen(mData) : 0`

Reply:
indeed but now the value of s is also checked, to get it more monkey proof